templates/module/helfi_tpr/tpr-service.html.twig

```diff
@@ -37,7 +37,17 @@
 
 {% elseif view_mode == 'teaser' %}
 
-  <div class="service service--teaser">
+  {%
+    set service_classes = [
+      'service',
+      'service--teaser',
+      not service_published ? 'service--teaser--unpublished',
```

Comment:
This variable could be more generic. Like `not content_entity_published ? 'service--teaser--unpublished',`

Reply:
Made it generic 🦖